packages/royalties/contracts/RoyaltySplitter.sol

```diff
@@ -0,0 +1,209 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {AddressUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/AddressUpgradeable.sol";
+import {ERC165Upgradeable} from "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
+import {SafeMath} from "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {BytesLibrary} from "@manifoldxyz/royalty-registry-solidity/contracts/libraries/BytesLibrary.sol";
+import {
+    IRoyaltySplitter,
+    IERC165,
+    Recipient
+} from "@manifoldxyz/royalty-registry-solidity/contracts/overrides/IRoyaltySplitter.sol";
+import {IRoyaltyManager} from "./interfaces/IRoyaltyManager.sol";
+import {IERC20Approve} from "./interfaces/IERC20Approve.sol";
+
+/**
+ * Cloneable and configurable royalty splitter contract
+ */
+contract RoyaltySplitter is Initializable, OwnableUpgradeable, IRoyaltySplitter, ERC165Upgradeable {
+    using BytesLibrary for bytes;
+    using AddressUpgradeable for address payable;
+    using AddressUpgradeable for address;
+    using SafeMath for uint256;
+
+    uint256 internal constant Total_BASIS_POINTS = 10000;
+    uint256 internal constant IERC20_APPROVE_SELECTOR =
+        0x095ea7b300000000000000000000000000000000000000000000000000000000;
+    uint256 internal constant SELECTOR_MASK = 0xffffffff00000000000000000000000000000000000000000000000000000000;
+
+    address payable public _recipient;
+    IRoyaltyManager public _royaltyManager;
+
+    event ETHTransferred(address indexed account, uint256 amount);
+    event ERC20Transferred(address indexed erc20Contract, address indexed account, uint256 amount);
+
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        virtual
+        override(IERC165, ERC165Upgradeable)
+        returns (bool)
+    {
+        return interfaceId == type(IRoyaltySplitter).interfaceId || super.supportsInterface(interfaceId);
+    }
+
+    /**
```

Comment:
Can you make natspec consistent
Our Sandbox contracts should look consistent across the package